samples/sqltransport/multi-instance/EndpointConnectionStringLookup/EndpointConnectionLookup.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace UserImplementation
```

Comment:
I believe it's useful to identify this class as code within the sample that the customer would write.